src/content/app/tools/blast/views/blast-submission-results/BlastSubmissionResults.tsx

```diff
@@ -32,8 +59,180 @@ const BlastSubmissionResults = () => {
   );
 };
 
+const Header = (props: Props) => {
```

Comment:
This is an exact copy of the `Header` defined in `ListedBlastSubmission` except for optionally displaying the action icons. Can it be moved to the `components` folder?